layouts/partials/post.html

```diff
@@ -24,8 +24,12 @@ <h1>{{ .Title | markdownify }}</h1>
 	{{ end }}
 
 	{{ if $displayDate }}
-          <p class="post-date">{{ dateFormat (or .Site.Params.dateFormat "January 2, 2006") .Date}}
-           {{ if lt .Date .Lastmod }} | Updated {{ dateFormat (or .Site.Params.dateFormat "January 2, 2006") .Lastmod }}{{ end }}
+          <p class="post-date">
+              {{ .ReadingTime }} min read | {{ dateFormat (or .Site.Params.dateFormat "January 2, 2006") .Date }}
```

Comment:
Hi @jamesericdavidson can you please add it as a feature flag? And keep the default value false. To prevent displaying a lot of info to the user on this page.